index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>YouTube Player Page</title>
+        <!-- Load icon library -->
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link href="styles/styles.css" rel="stylesheet" type="text/css" />
+        <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+    </head>
+    <body>
+        <header>
+                <div id="header_container">
+                    <div class="fornavis">
+                        <a class="navis" href="navi.html">
+                            <i class="fa fa-bars"></i>
+                        </a>
+                    </div>
+                    
+                    <div class="forimg">
+                        <a class="yicon" href="https://www.youtube.com">
+                            <img class="imgspan" src="Images/yicon.png" width="35" height="35" />
```

Comment:
Kindly put the `alt` attribute to your `img` tag. This is the best practice to follow when using the `img` tag as this will help screen readers to know more about your page